src/components/browse/dataset/datasetAccordion/datasetSummary/datasetSummary.tsx

```diff
@@ -29,7 +29,7 @@ const DatasetSummary = (props: dataSetDetailsProps) => {
   }*/
 
   const handleOpen = () => {
-    setCopyEmail(props.details !== null ? props.details?.dac_email : "helpdesk@ghga.de");
+    setCopyEmail(props.details !== null && props.details ? props.details.dac_email : "helpdesk@ghga.de");
     setShow(true);
```

Comment:
do we need the getDACEmailId method now?

Reply:
That was your code. Out of scope of this PR.